web/src/views/application/components/ApplicationForm.vue

```diff
@@ -20,7 +20,7 @@
 import { useFormEvent } from '@/composables'
 
 export default defineComponent({
-  name: 'ApplicationSearchForm',
```

Comment:
It's better to create another follow up pull request for unifying naming of applicaiton module.